live-examples/js-examples/intl/meta.json

```diff
@@ -78,6 +78,12 @@
             "title": "JavaScript Demo: Intl.GetCanonicalLocales",
             "type": "js"
         },
+        "intlListFormatPrototypeResolvedOptions": {
+            "exampleCode": "./live-examples/js-examples/intl/intl-listformat-prototype-resolvedoptions.js",
+            "fileName": "intl-listformat-prototype-resolvedoptions.html",
+            "title": "JavaScript Demo: Intl.ListFormat.prototype.resolvedOptions()",
+            "type": "js"
+        },
```

Comment:
this should go after "intlListFormat" (really we should have CI that checks these JSON files are alphabetised).